account_move_line_sequence/models/account_move_line.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+    _order = "sequence, date desc, name desc, id"
```

Comment:
Whoofff... in its current form this is going to annoy quite a number of instances with a lot of move lines.

Before OCA-ing this further, I would like to know what his idea with the sequence is.. is it to order things within one journal entry or does he want to order the whole lot of journal entries this way? And in the latter case, maybe only the Journal entries view? Then we could rather put this sorting on that one, and not burden the rest of the system.

Let me ask him.

Reply:
@ntsirintanis it's indeed meant to only provide sorting on the move's form view so we don't want to make anything else less performant.